android/src/app/organicmaps/MwmActivity.java

```diff
@@ -1039,13 +1074,13 @@ protected void onStart()
     BookmarkManager.INSTANCE.addLoadingListener(this);
     RoutingController.get().attach(this);
     IsolinesManager.from(getApplicationContext()).attach(this::onIsolinesStateChanged);
-    LocationHelper.INSTANCE.attach(this);
     LocationState.nativeSetListener(this);
     LocationHelper.INSTANCE.addListener(this);
     onMyPositionModeChanged(LocationState.nativeGetMode());
     mSearchController.attach(this);
     if (!Config.isScreenSleepEnabled())
       Utils.keepScreenOn(true, getWindow());
+    LocationState.nativeSetLocationPendingTimeoutListener(this::onLocationPendingTimeout);
```

Comment:
Shouldn't it be subscribed in onResume/onPause?

Reply:
Moved to onResume/onPause.